app/views/home/contributors.html.erb

```diff
@@ -1,15 +1,15 @@
 <div class="bg-gray mb-6 border-bottom jumbotron px-3">
   <div class="container-md">
     <h2 class="alt-h1 text-center text-blue">Make a contribution</h2>
-    <p class="alt-lead text-center text-gray">Open source software is made by people just like you. Take some time one day every week and make your contribution.
+    <p class="alt-lead text-center text-gray">Open source is made by people just like you. Once a week, contribute back to the software you use and love.
```

Comment:
@bkeepers realtalk: I wanted to say "This week" but then your pedantic linter rang through my head :laughing: